src/settings.rs

```diff
@@ -272,47 +173,33 @@ mod tests {
     }
 
     #[test]
-    fn settings_serialization_test() -> Result<(), ()> {
+    fn settings_serialization_test() {
```

Comment:
I wonder why we had this test in the first place, looks like testing serde... I would drop it